simple_history/manager.py

```diff
@@ -121,3 +121,35 @@ def bulk_history_create(self, objs, batch_size=None):
         return self.model.objects.bulk_create(
             historical_instances, batch_size=batch_size
         )
+
```

Comment:
All of this may be unnecessary, I think it may be a full copy-paste of the other descriptor in here since I've removed all of the other functionality that lived here.